client/src/components/Navbar.vue

```diff
@@ -241,31 +173,28 @@ import { useColorPalette, useDialog } from '../stores/useUserInterfaceStore'
 import { useGoogleUserData } from '../stores/useDataStore'
 
 import TestComponent from './TestComponent.vue'
+import UserAuthMenu from './UserAuthMenu.vue'
 
 const { smAndDown, mdAndUp, smAndUp, xs } = useDisplay()
 const { gray, blue, white, textPrimary, textSelected } = useColorPalette()
 const { getGoogleUserData, redirectToGoogleAuth, logUserOut, logUserIn } = useGoogleUserData()
 const { showDialog } = useDialog()
 
-const checkIfKeepLoggedIn = () => {
-  try {
-    const googleProfileData = localStorage.getItem('googleProfileData')
-    return googleProfileData !== null
-  } catch (error) {
-    console.error('Error retrieving variable from localStorage:', error)
-    return false;
-  }
-}
 
-const keepUserLoggedIn = ref(checkIfKeepLoggedIn())
+const keepUserLoggedIn = ref(!localStorage.getItem('googleProfileData'))
```

Comment:
what is the point of this if its not being referenced anywhere else?

also local storage should always be a try catch since some devices do not have localstorage